engine-precompiles/src/native.rs

```diff
@@ -30,12 +30,10 @@ mod costs {
     // TODO(#483): Determine the correct amount of gas
     pub(super) const EXIT_TO_ETHEREUM_GAS: EthGas = EthGas::new(0);
 
-    // TODO(#332): Determine the correct amount of gas
-    pub(super) const FT_TRANSFER_GAS: NearGas = NearGas::new(100_000_000_000_000);
+    pub(super) const FT_TRANSFER_GAS: NearGas = NearGas::new(10_000_000_000_000);
```

Comment:
I don't want to block this PR, but I do believe that rationale for this value should be contributed. Same for the one blow.